x-pack/plugin/redact/src/main/java/org/elasticsearch/xpack/redact/RedactProcessor.java

```diff
@@ -341,6 +374,7 @@ public RedactProcessor create(
             String matchField = ConfigurationUtils.readStringProperty(TYPE, processorTag, config, "field");
             List<String> matchPatterns = ConfigurationUtils.readList(TYPE, processorTag, config, "patterns");
             boolean ignoreMissing = ConfigurationUtils.readBooleanProperty(TYPE, processorTag, config, "ignore_missing", true);
+            boolean skipIfUnlicensed = ConfigurationUtils.readBooleanProperty(TYPE, processorTag, config, "skip_if_unlicensed", true);
```

Comment:
I think I'd prefer to default this to `false`, what do you think?

Reply:
Yeah, for sure, I was on the fence about what the right default value is. I'm happy to change it.

Reply:
Yeah I'd think that for something that we advertise as being usable to redact PII, we wouldn't want it to silently no-op if someone temporarily mixed up a license.